setup.py

```diff
@@ -202,6 +202,11 @@ def run(self):
     :undoc-members:
 """ % mod)
 
+# (function, predicate), see http://docs.python.org/2/distutils/apiref.html#distutils.cmd.Command.sub_commands
```

Comment:
why did you add this again? it was not possible to install without building docs.